vars/debugReportArchive.groovy

```diff
@@ -0,0 +1,65 @@
+import com.sap.piper.ConfigurationHelper
+import com.sap.piper.DebugReport
+import com.sap.piper.GenerateDocumentation
+import com.sap.piper.Utils
+import groovy.transform.Field
+
+import static com.sap.piper.Prerequisites.checkScript
+
+@Field def STEP_NAME = getClass().getName()
+@Field Set GENERAL_CONFIG_KEYS = []
+@Field Set STEP_CONFIG_KEYS = [
+    /**
+     * Flag to control whether confidential information (like GitGub credentials) are to be included in the
```

Comment:
For which reason should we log github credentials? 
I assume we do not log it and the example is just not correct?

Reply:
You are absolutely right. No clue how I arrived at this impression. I've now enumerated all information that is included with this flag in its documentation.